src/components/Tour.vue

```diff
@@ -44,11 +44,45 @@ export default {
 				},
 				'ide-workspace': {
 					title: 'User Workspace',
-					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services and files.'
+					content: 'In this area you can find your personal workspace, which lists your stored batch jobs, web services, custom processes and files.'
 				},
 				'ide-viewer': {
 					title: 'Data Viewer',
-					content: 'This area you can use to preview collections or inspect the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+					content: 'This area is for previewing collections or inspecting the results of batch jobs, web services or other computations. It will also be used to display log messages, if available.'
+				},
+
+				'connect-url': {
+					title: 'URL input',
+					content: `Paste the URL of the server you wish to connect to into this field. If you don't know any URL, you may head over to <a href="https://hub.openeo.org/">openEO Hub</a> to discover available servers.`
+				},
+				'connect-history': {
+					title: 'History of URLs',
+					content: 'Clicking on this icon will open a window that displays all URLs that you have connected to in the past (provided your browser permits storing in Local Storage).'
+				},
+
+				'login-options': {
+					target: '#credentials > .tabsHeader',
+					title: 'Login Options',
+					content: `You have several ways of authenticating with the server. The standard method is OpenID Connect (OIDC). If supported, you will see a button for each available OIDC provider here. Click on one to open its login form.`
+				},
+				'login-internal': {
+					target: '#credentials > .tabsHeader > button[title=Internal]',
+					title: 'User/Password',
+					content: `Some servers support basic authentication with a simple username-password combination. This method is discouraged and generally only used for internal development purposes (hence the name of this button). We suggest to use OpenID Connect if you can.`
+				},
+				'login-without': {
+					target: '#credentials > .tabsHeader > button[title="No credentials"]',
+					title: 'Without login',
+					content: `It is possible to proceed without any authentication, however, usually this means that you won't able to actually submit calculations etc. You can think of it as an "discovery only" mode.`
+				},
+				'login-credentials': {
+					target: '.tabContent:not([style*="display: none"]) .input',  // select first visible input
+					title: 'Credentials of your account',
+					content: `Enter your authentication information here. If you don't have an account yet, you have to register directly with the provider (this is not possible via openEO). The same applies if you forgot your password.`
+				},
+				'login-switch': {
+					title: 'Wrong server?',
+					content: `If you chose the wrong server, this link takes you back to the previous step so you can choose a different server URL.`
```

Comment:
```suggestion
					content: `If you chose the wrong server, this link takes you back to the previous step so you can choose another server.`
```

Reply:
To me, "another" sounds more like "one more", therefore changing back.

```suggestion
					content: `If you chose the wrong server, this link takes you back to the previous step so you can choose a different server.`
```